app/assets/stylesheets/components/rad-notification/_settings.scss

```diff
@@ -0,0 +1,5 @@
+$rad-notification-radius: 5px;
+$color-rad-notification-warning: #F89E54;
```

Comment:
Could this reference another variable in the main palette for RAD? Could potentially be reused?